db/migrate/20200126203714_remove_vim_types_from_ems_events.rb

```diff
@@ -0,0 +1,27 @@
+class RemoveVimTypesFromEmsEvents < ActiveRecord::Migration[5.1]
+  disable_ddl_transaction!
+  include MigrationHelper
+
+  class EventStream < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    say_with_time("Removing Vim Types from EmsEvents") do
+      base_relation = EventStream.in_my_region.where(:source => "VC")
```

Comment:
If the type column is indexed, we can probably add that into the where clause as well (might even be better than having source)

Reply:
Actually neither of them do?
```
>> EventStream.connection.index_exists?(:event_streams, :type)
=> false
>> EventStream.connection.index_exists?(:event_streams, :source)
=> false
```

Reply:
:(